src/noit_metric_director.h

```diff
@@ -35,12 +35,14 @@
 #include <mtev_hooks.h>
 #include <mtev_uuid.h>
 #include <noit_message_decoder.h>
+#include <noit_metric_tag_search.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-typedef unsigned short caql_cnt_t;
+typedef unsigned short interest_cnt_t;
```

Comment:
if this is unsigned why are there tests in the implementation file checking if it's under 0?  The value would need to be cast or checked using bits to see if it's in the negative range

Reply:
We are applying checks to a larger type to ensure it doesn't roll.